components/src/core/components/CardTable/Decorator/DefaultCardWeb.vue

```diff
@@ -68,7 +68,23 @@ export default defineComponent({
   },
 
   methods: {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    bindClasses(index: number) {
```

Comment:
please remove this if you agree with other comment

Reply:
Removed